### DISCUSSION TITLE
Wrappers para diversas linguagens

### DISCUSSION THREAD

**Discussion by rfoel** [BrasilAPI/BrasilAPI#173] [Show and tell]
Eu acabei criando um cliente em JavaScript pro BrasilAPI [aqui](https://github.com/rfoel/brasilapi) e gostaria de compartilhar a ideia.

Eu sempre preferi usar uma biblioteca que fizesse o wrapper de uma REST api, assim eu tenho em mão todos os métodos disponíveis facilmente, e com typescript eu sei os parâmetros e posso interagir com a resposta esperada, sabendo todos os types.

A ideia é criar outros clientes em Python, Go, etc. O que acham?

**Comment by RodriAndreotti** [comment]
Eu acho uma ideia interessante, mas estou pensando que isso no futuro ficará difícil de manter sincronizado com a API, a medida que novos endpoints forem implementados , os mantenedores dos wrappers teriam que disponibilizar as novas rotas tbm.

Mas na minha humilde opinião, creio que é uma ideia que podemos amadurecer sim.
